libs/MVS/SceneTexture.cpp

```diff
@@ -509,11 +519,36 @@ bool MeshTexture::ListCameraFaces(FaceDataViewArr& facesDatas, float fOutlierThr
 		faceMap.create(imageData.height, imageData.width);
 		depthMap.create(imageData.height, imageData.width);
 		RasterMesh rasterer(vertices, imageData.camera, depthMap, faceMap);
+		//Creating mask for the image border
```

Comment:
pls use the same stile as the rest of the code: comments inside functions start with small letter, and a space between // and word